### PR TITLE
Feature/160 option all

### PR DESCRIPTION
**Pull request by ThunderbirdOne**

### CODE REVIEW

**Review by VandeVelGianni — APPROVED** [review]
I'm not going to block changes, but I'd tag the user if they (un)sub to all lists, so it's consistent with the rest of the notify-commands. Image for clarity.
image

Rest looks good to me :)

**Comment by Autom3** [comment]
If #140 gets merged before this one, a small edit is required in the translation file

**Comment by VandeVelGianni** [comment]
> If #140 gets merged before this one, a small edit is required in the translation file

Same in reverse if this is before #140 .

File: nerdlandbot/translations/Translations.csv [hunk 1]
```diff
@@ -157,3 +158,5 @@
 "poncho_pic_help","Post a random Poncho picture.","Post een willekeurige foto van Poncho."
 "poncho_hond","DOG DOG DOG","HOND HOND HOND"
 
+"all_sub_success","You are now subbed to every list","Je bent nu geabboneerd op elke lijst"
+"all_unsub_success","You are no longer subscribed to any lists","Je bent nu op geen enkele lijst meer geabboneerd"
```
Comment (Eva774) [inline nerdlandbot/translations/Translations.csv:162]: Isn't there a thing that we should leave a newline at the end of every file?

**Review by Eva774 — APPROVED** [review]

**Pull request merged**